packages/jobs/src/bins/__tests__/rw-jobs-worker.test.js

```diff
@@ -0,0 +1,9 @@
+import { describe, expect, it } from 'vitest'
+
+// import * as worker from '../worker'
+
+describe('worker', () => {
```

Comment:
Do we want to have unit tests here? I'm happy with the answer being no if this isn't a sensible place for unit testing and e2e style testing would be better?